Projects/Playground/Playground.Forms.iOS/AppDelegate.cs

```diff
@@ -11,22 +12,15 @@ public partial class AppDelegate : MvxFormsApplicationDelegate
     {
         public override bool FinishedLaunching(UIApplication app, NSDictionary options)
         {
-            Window = new UIWindow(UIScreen.MainScreen.Bounds);
-
             UINavigationBar.Appearance.BarTintColor = UIColor.FromRGB(65, 105, 225);
             UINavigationBar.Appearance.TintColor = UIColor.FromRGB(255, 255, 255);
 
-            var setup = new Setup(this, Window);
-            setup.Initialize();
-
-            var startup = Mvx.Resolve<IMvxAppStart>();
-            startup.Start();
-
-            LoadApplication(setup.FormsApplication);
-
-            Window.MakeKeyAndVisible();
-
             return base.FinishedLaunching(app, options);
         }
+
+        protected override MvxIosSetup CreateSetup(IMvxApplicationDelegate applicationDelegate, UIWindow window)
```

Comment:
We should probably move this into the normal appdelegate as well.

Reply:
Done